backend/src/memberships/entities/membership.entity.ts

```diff
@@ -26,12 +26,14 @@ export class Membership {
   @ManyToOne(() => Channel)
   channel: Channel;
   @RelationId((membership: Membership) => membership.channel)
+  @Column()
   channelId: number;
 
   @Exclude()
   @ManyToOne(() => User)
```

Comment:
```suggestion
  @ManyToOne(() => User, { onDelete: 'CASCADE' })
```
Otherwise a user cannot be deleted if a membership exists for that user